adapters/repos/db/lsmkv/bucket.go

```diff
@@ -1615,10 +1625,16 @@ func (b *Bucket) CreateDiskTerm(N float64, filterDocIds helpers.AllowList, query
 		if err != nil {
 			return nil, lock, err
 		}
-		allTombstones[j] = sroar.Or(allTombstones[j+1], tombstones)
+
+		if b.flushing != nil {
+			tombstones.SetMany(allTombstones[0])
+		}
+		if b.active != nil {
+			tombstones.SetMany(allTombstones[1])
```

Comment:
`tombstones.Or(allTombstones[1])` // provided allTombstones are bitmaps, not arrays

Reply:
bitmaps are not safe to be used concurrently, unless some external locking mechanism is used